src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -80,6 +82,7 @@ String processClassPath(String cp) throws IOException {
     @Override
     public boolean compile(String cp) throws IOException, InterruptedException {
         String classPath = processClassPath(cp);
+        extractNativeLibs(cp);
```

Comment:
why do we need this at compiletime?

Reply:
We need to parse the classpath. This could be done in the link phase, but that will require a change in the API to pass the classpath again.

Reply:
makes sense. I see the extracted libs are stored in a specific path (which is then the contract between compile and link phase). I don't think that is bad (at least I have no better solution).